projects/resupply/index.js

```diff
@@ -0,0 +1,78 @@
+const sdk = require('@defillama/sdk');
+const { getConfig } = require('../helper/cache');
+
+
+const convertToAssetsAbi = 'function convertToAssets(uint256) view returns (uint256)';
+const coinsAbi = 'function coins(uint256) view returns (address)';
+const balancesAbi = 'function balances(uint256) view returns (uint256)'
+
+async function tvl(api) {
+    const contracts = await getConfig('resupply/tvl/', 'https://raw.githubusercontent.com/resupplyfi/resupply/main/deployment/contracts.json');
+
+    // Lending Market
+    const pairsContracts = Object.entries(contracts)
+        .filter(([key]) =>
+        !key.endsWith('_DEPRECATED') &&
```

Comment:
we are fine if something is deprecated, we count it as tvl as long as there are user funds in the contract